### DISCUSSION TITLE
My Skycrypt shows Haste ring in missing accessory, but i have haste Artifact already.

### DISCUSSION THREAD

**Discussion by nobprog** [SkyCryptWebsite/SkyCrypt#2298] [General]
Hello i'm new to Github and making discussion in here.
I think this is a bug, or is Haste ring and Haste artifact are not same "upgrade route"?

**Comment by DuckySoLucky** [comment]
Hello, yes they are. SkyCrypt's accessories r currently outdated

**Reply by nobprog** [reply]
i hope they will fix it soon, or should i make a "pull request"?
idk how to make one

**Reply by DuckySoLucky** [reply]
expect a fix within 2-3 business days
